include/avif/avif.h

```diff
@@ -6,6 +6,7 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#include <stdlib.h> // for strtol(3)
```

Comment:
Thank you for the pull request. The only file that uses strtol() is contrib/gdk-pixbuf/loader.c. Please move this #include <stdlib.h> statement to that file. I also suggest omitting the "for strtol(3)" comment. That kind of comment is not commonly used in libavif. Thanks!